README.md

```diff
@@ -19,7 +19,7 @@ Error mitigation seeks to reduce these effects at the software level by
 compiling quantum programs in clever ways.
 
 Want to know more? Check out our
-[documentation](https://mitiq.readthedocs.io/en/stable/guide/guide.html), chat with us on [Discord](http://discord.unitary.fund), and join our weekly community call ([public agenda](https://docs.google.com/document/d/1lZfct4AOCS7fdyWkudcGyER0n0nsCxSFKSicUEeJgtA/)).
+[documentation](https://mitiq.readthedocs.io/en/stable/guide/guide.html). For code, repo, or theory questions, especially those requiring more detailed responses, submit a [Discussion](https://github.com/unitaryfund/mitiq/discussions). For time sensitive questions, chat with us on [Discord](http://discord.unitary.fund). Also, join our weekly community call ([public agenda](https://docs.google.com/document/d/1lZfct4AOCS7fdyWkudcGyER0n0nsCxSFKSicUEeJgtA/)).
```

Comment:
Thanks for adding more context. Just to make sure it's readable, could we split into bullet points perhaps?
- docs
- discussions
- time sensitive or casual

Reply:
I split it into bullet points and also added `on Discord` for the weekly community call.